### PR TITLE
automate OCP-63777 & OCP-63134

### PR DESCRIPTION
**Pull request by shellymiron**
additional post day 1 test cases: [OCP-63777](https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-63777) & [OCP-63134](https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-63134)

image

### CODE REVIEW

File: tests/e2e/verfication_post_day1_test.go [hunk 1]
```diff
@@ -61,22 +78,34 @@ var _ = Describe("TF Test", func() {
...
+		})
+		Context("Author:smiron-High-OCP-63777 @OCP-63777 @smiron", func() {
+			It("Verify AWS tags are set post cluster deployment", CI.Day1Post, CI.High, func() {
+				getResp, err := CMS.RetrieveClusterDetail(CI.RHCSConnection, clusterID)
+				Expect(err).ToNot(HaveOccurred())
+
+				if profile.Tagging {
+					Expect(getResp.Body().AWS().Tags()["tag1"]).To(Equal(CON.Tags["tag1"]))
+				} else {
+					Expect(getResp.Body().AWS().Tags()["tag1"]).To(Equal(""))
```
Comment (amalykhi) [inline tests/e2e/verfication_post_day1_test.go:108]: @shellymiron did you verify non-tagging scenario? I guess tag1 shouldn't be found or return null/error

Comment (shellymiron) [reply tests/e2e/verfication_post_day1_test.go:108]: yeah, checkout cluster `export CLUSTER_ID=27c2i17u3f3i02kub55ng4639en0c6ed`       
image

Comment (shellymiron) [reply tests/e2e/verfication_post_day1_test.go:108]: I think that in golang if a map's key is not found, it just return empty string, that's why the test worked

File: tests/e2e/verfication_post_day1_test.go [hunk 2]
```diff
@@ -61,22 +78,34 @@ var _ = Describe("TF Test", func() {
...
 		Context("Author:smiron-High-OCP-68423 @OCP-68423 @smiron", func() {
 			It("Verify compute_labels are set post cluster creation", CI.Day1Post, CI.High, func() {
 				getResp, err := CMS.RetrieveClusterDetail(CI.RHCSConnection, clusterID)
 				Expect(err).ToNot(HaveOccurred())
+
 				if profile.Labeling {
 					Expect(getResp.Body().Nodes().ComputeLabels()).To(Equal(CON.DefaultMPLabels))
 				} else {
-					Expect(getResp.Body().Nodes().ComputeLabels()).To(Equal(""))
+					Expect(getResp.Body().Nodes().ComputeLabels()).To(Equal(CON.NilMap))
```
Comment (xueli181114) [inline tests/e2e/verfication_post_day1_test.go:96]:
```suggestion
					Expect(getResp.Body().Nodes().ComputeLabels()).To(BeNil())
```

File: tests/utils/constants/constants.go [hunk 3]
```diff
@@ -37,9 +37,11 @@ var (
 	GitLabURL           = "https://gitlab.cee.redhat.com"
 	Organizations       = []string{"openshift"}
 	HostedDomain        = "redhat.com"
+	NilMap              map[string]string
```
Comment (xueli181114) [inline tests/utils/constants/constants.go:40]: I don't think we need this

File: tests/e2e/verfication_post_day1_test.go [hunk 4]
```diff
@@ -61,22 +78,34 @@ var _ = Describe("TF Test", func() {
...
+				}
+			})
+		})
+		Context("Author:smiron-High-OCP-63777 @OCP-63777 @smiron", func() {
+			It("Verify AWS tags are set post cluster deployment", CI.Day1Post, CI.High, func() {
+				getResp, err := CMS.RetrieveClusterDetail(CI.RHCSConnection, clusterID)
+				Expect(err).ToNot(HaveOccurred())
+
+				if profile.Tagging {
+					Expect(getResp.Body().AWS().Tags()["tag1"]).To(Equal(CON.Tags["tag1"]))
```
Comment (xueli181114) [inline tests/e2e/verfication_post_day1_test.go:106]:
```suggestion
                                        Expect(len(getResp.Body().AWS().Tags())).To(Equal(len(CON.Tags)))
                                        for k,v := range getResp.Body().AWS().Tags(){
                                        Expect(v).To(Equal(CON.Tags[k]))
                                        }
					
```

Comment (xueli181114) [reply tests/e2e/verfication_post_day1_test.go:106]: Don't hardcode it.

File: tests/utils/constants/constants.go [hunk 5]
```diff
@@ -37,9 +37,11 @@ var (
...
 	DefaultMPLabels     = map[string]string{
 		"test1": "testdata1",
 	}
+	Tags = map[string]string{"tag1": "test_tag1"}
```
Comment (xueli181114) [inline tests/utils/constants/constants.go:44]: Better to set multiple tags

File: tests/e2e/verfication_post_day1_test.go (hunk 2 above)
Comment (shellymiron) [reply tests/e2e/verfication_post_day1_test.go:96]: I changed to `BeNil` and reciving this err:
```
  [FAILED] Expected
      <map[string]string | len:0>: nil
  to equal
      <*matchers.BeNilMatcher | 0x1044caf40>: {}
```
it seems like even the nil of the empty map, is of type `string`

File: tests/utils/constants/constants.go (hunk 5 above)
Comment (shellymiron) [reply tests/utils/constants/constants.go]: done

File: tests/e2e/verfication_post_day1_test.go (hunk 4 above)
Comment (shellymiron) [reply tests/e2e/verfication_post_day1_test.go]: fixed

File: tests/ci/profiles/tf_cluster_profile.yml [hunk 6]
```diff
@@ -48,7 +48,7 @@ profiles:
     compute_machine_type: "m5.2xlarge"
     proxy: true
     labeling: true
-    tagging: true
+    tagging: false
```
Comment (amalykhi) [inline tests/ci/profiles/tf_cluster_profile.yml:51]: why false?

Comment (shellymiron) [reply tests/ci/profiles/tf_cluster_profile.yml]: opsss, you are right. leftover from testing
fixed

**Pull request merged**